### ISSUE TITLE
Streams are not greyed out

### ISSUE THREAD

**Issue by anirudhjain75** [zulip/zulip-android#292]
Streams in Zulip web are greyed out but in the app they are not

**Comment by anirudhjain75** [comment]
![screen shot 2017-01-09 at 4 03 10 pm](https://cloud.githubusercontent.com/assets/14249589/21763694/29076b02-d685-11e6-90a9-de05be57b134.png)Desktop Version

**Comment by anirudhjain75** [comment]
![screenshot_20170109-160358](https://cloud.githubusercontent.com/assets/14249589/21763762/88e0ab6a-d685-11e6-9978-4e8eed3f2fa9.png)
Android version

**Comment by TomaszKolek** [comment]
@Sam1301 tried to reproduce it on newest version of the app - and she cannot reproduce it.

@anirudhjain75 Could you update the app and try once again?

**Comment by anirudhjain75** [comment]
@TomaszKolek I have the latest version on android

**Comment by TomaszKolek** [comment]
So could you please try once again? 
@anirudhjain75

**Comment by anirudhjain75** [comment]
[screenrecorder.20170109172943.mp4.zip](https://github.com/zulip/zulip-android/files/693364/screenrecorder.20170109172943.mp4.zip)
@Sam1301 @TomaszKolek

**Comment by Sam1301** [comment]
Hey @anirudhjain75 , as we just found here https://chat.zulip.org/#narrow/near/120578/stream/android/topic/bug.20hunt, this greying is not related to streams being muted :)

**Comment by anirudhjain75** [comment]
What is the other reason streams get muted

**Comment by anirudhjain75** [comment]
The other reason they are greyed out is `not being used for long` as mentioned by @trueskawka

**Comment by showell** [comment]
Is it possible that the stream is grayed out due to being inactive for a long time?

**Comment by Sam1301** [comment]
I agree, @anirudhjain75 and I had a chat in android stream, and found out that this graying is due to streams not active for long which is currently not accounted for in the app.
